components/bl-parallax-component/src/index.js

```diff
@@ -20,7 +26,7 @@ export default function Parallax({ component, pods }) {
       <div ref={ backdropRef } className="parallax-background-img"
            style={ { backgroundImage: `url(${ imageUrl })` } }></div>
       <div className="parallax-content">
-        { parallaxContentPod.render() }
+        { parallaxContentPod && parallaxContentPod.render() }
```

Comment:
Could it be like that?

Approach with empty catch looks bad for me.

```suggestion
        { pods['parallaxContent']?.render() }
```